includes/ting_search_autocomplete.admin.inc

```diff
@@ -0,0 +1,210 @@
+    '#size' => 25,
+    '#default_value' => $default_settings['index'],
+    '#required' => TRUE,
+  );
+
+  $form['ting_search_autocomplete_settings']['facetIndex'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Facet Index', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 25,
+    '#default_value' => $default_settings['facetIndex'],
+  );
+
+  $form['ting_search_autocomplete_settings']['filterQuery'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Filter Query', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 25,
+    '#default_value' => $default_settings['filterQuery'],
+  );
+
+  $form['ting_search_autocomplete_settings']['sort'] = array(
+    '#type' => 'radios',
+    '#title' => t('Output sort order', array(), array('context' => 'ting_search_autocomplete')),
+    '#options' => array('count' => 'count', 'index' => 'index',),
+    '#default_value' => $default_settings['sort'],
+  );
+
+  $form['ting_search_autocomplete_settings']['agency'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Agency', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['agency'],
+  );
+
+  $form['ting_search_autocomplete_settings']['profile'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Profile', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['profile'],
+  );
+  $form['ting_search_autocomplete_settings']['maxSuggestions'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max suggestions', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxSuggestions'],
+  );
+
+  $form['ting_search_autocomplete_settings']['maxTime'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max request time', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxTime'],
+  );
+
+  $form['ting_search_autocomplete_settings']['highlight'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Highlight', array(), array('context' => 'ting_search_autocomplete')),
+    '#description' => t('If checked, search prefix will be enclosed in highlight prefix and suffix', array(), array('context' => 'ting_search_autocomplete')),
+    '#default_value' => $default_settings['highlight'],
+  );
+
+  $form['ting_search_autocomplete_settings']['highlightPre'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight prefix', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 20,
+    '#default_value' => $default_settings['highlightPre'],
+  );
+
+  $form['ting_search_autocomplete_settings']['highlightPost'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight suffix', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 18,
```

Comment:
The size of these elements vary wildly - e.g. `highlightPre` is 20 but `highlightPost` is only 18. Is there a reason why it is so?